tests/dummy/app/reducers/combined.js

```diff
@@ -0,0 +1,10 @@
+var combined = function combined(state, action) {
+  if (action.type === 'COMBINE') {
+    // so we fail the patch reducers test if the
+    // patch fails/ or didn't apply as expected
+    // return Object.assign({}, state, { done: true });
```

Comment:
It might be better to throw an explicit `Ember.assert` in here if this is invoked to make it clearer to someone reading this code in the future.